e0works/MyDiary-front
# #162359721 deploy base application to heroku

#### What does this PR do?
- deploy the base app to Heroku

#### Description of Task to be completed?
- add app.js to server our index
- add deploy block to .travis.yml

#### How should this be manually tested?
- run `npm run build`
- run `npm start`
- you should see the app running in production mode on port `8002`
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#162359721](https://www.pivotaltracker.com/story/show/162359721, "deploy the base app to Heroku")


#### Screenshots (if appropriate)
NA

#### Questions:
